nuxeo-jsf/nuxeo-platform-webapp-base/src/main/resources/web/nuxeo.war/widgets/listing/listing_lock_icon_widget_template.xhtml

```diff
@@ -4,6 +4,7 @@
   xmlns:f="http://java.sun.com/jsf/core"
   id="#{widget.id}"
   rendered="#{not empty field and not empty field_1}">
-  <f:attribute name="title" value="#{messages['label.summary.by']} #{nxu:userFullName(field_1.owner)}, #{messages['label.summary.the']} #{field_1.created.time}" />
+
```

Comment:
Unneeded extra empty line.

Reply:
Sorry, it's just the habit i have to always add an empty line in the end of the file